app/assets/stylesheets/home.scss

```diff
@@ -66,7 +66,6 @@ p.choose_party {
   background-color: white;
 
   .user {
-    display: inline-block;
```

Comment:
I guess these changes are deliberate but not sure how they relate to rationalising variables - were they supposed to be in a different commit?

Reply:
Sorry, my commits weren't brilliantly distinct. I will try and describe things better in future.